src/duke/Storage.java

```diff
@@ -0,0 +1,138 @@
+package duke;
+
+import java.io.*;
+import java.util.ArrayList;
+
+
+/**
+ * This is an FileAction class that allows the saving of
+ * contents into a pre-specified file
+ *
+ * @author  Hsiao Jiet
+ * @version 1.0
+ * @since   2022-1-25
+ */
+
+public class Storage {
+    protected String filepath;
+    protected FileWriter fileWriter;
+    protected FileReader fileReader;
+    protected BufferedWriter writer;
```

Comment:
Do you think it's better to name the BufferedWriter/Readers as buffWriter or buffReader, since there are similar fields for FileWriter/Reader?